src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Running a test from the CLI.md

```diff
@@ -0,0 +1,265 @@
+| ------------- | ------ | -------------------------------------------------------------------------------------------------------- |
+| `load_zone`   | string | The load zone from where the the metric was collected. Values will be of the form: `amazon:us :ashburn`. |
+| `instance_id` | number    | A unique number representing the ID of a load generator server taking part in the test.                  |
+
+![filter tags](/images/analysis-tab-cloud-tags.png 'Cloud execution tags')
+
+
+## Environment variables
+
+[Environment variables](/using-k6/environment-variables) set in the local terminal before executing k6 won't be forwarded to the k6 cloud service, and thus won't be available to your script when executing in the cloud.
+
+With cloud execution, you must use the CLI flags (`-e`/`--env`) to set environment variables like `-e KEY=VALUE` or `--env KEY=VALUE`.
+
+For example, given the script below, which reads the `MY_HOSTNAME` environment variable.
+
+
+<div class="code-group" data-props='{"labels": ["script.js"]}'>
+
+```javascript
+import { check, sleep } from 'k6';
+import http from 'k6/http';
+
+export default function() {
+  var r = http.get(`http://${__ENV.MY_HOSTNAME}/`);
+  check(r, {
+    'status is 200': r => r.status === 200,
+  });
+  sleep(5);
+}
+```
+
+</div>
+
+You'd execute it using the command like:
+
+
+<div class="code-group" data-props='{}'>
+
+```js
+$ k6 run -e MY_HOSTNAME=test.k6.io script.js
+```
+
+</div>
+
+### Injected environment variables on the cloud execution
+
+When running in the k6 Cloud there will be three additional environment variables that can be used to find out in which load zone, server instance, and distribution label the given script is currently running.
+
+Name|	Value|	Description
+-|-|-
+`LI_LOAD_ZONE`	|string|	The load zone from where the the metric was collected. Values will be of the form: amazon:us :ashburn (see list above).
+`LI_INSTANCE_ID`	|number|	A sequential number representing the unique ID of a load generator server taking part in the test, starts at 0.
+`LI_DISTRIBUTION`	|string|	The value of the "distribution label" that you used in `ext.loadimpact.distribution` corresponding to the load zone the script is currently executed in.
+
+You can read the values of these variables in your k6 script as usual.
+
+
+## Differences between local and cloud execution
+
+### Iterations
+
+Local execution has support for iterations based test length (`-i` or
+`--iterations` on CLI, and `iterations` in script options) which is
+not yet supported by the cloud execution mode.
+
+
+### Using setup/teardown life-cycle functions
+
+Your [setup and teardown life cycle functions](/using-k6/test-life-cycle)
+are executed as normal when running cloud tests. Depending on the size
+of your test, it will execute from one or more cloud servers, but the
+setup and teardown will only execute from one server, so execute once
+each test-wide. There's no guarantee though that the same cloud server
```

Comment:
"so execute once each test-wide" => "so execute once for each test run"

Reply:
https://github.com/loadimpact/k6-docs/pull/26/commits/63afdcc5f86e883180b9cb050c595edea88c4720